docs/docs/Configuration/configuration-custom-database.md

```diff
@@ -42,35 +40,141 @@ Replace the value for `LANGFLOW_DATABASE_URL` with your PostgreSQL connection st
 
 4. Run Langflow with the `.env` file:
 ```bash
-langflow run --env-file .env
+uv run langflow run --env-file .env
 ```
 
 5. In Langflow, create traffic by running a flow.
 6. Inspect your PostgreSQL deployment's tables and activity.
-You will see new tables and traffic created.
+New tables and traffic are created.
 
 ## Example Langflow and PostgreSQL docker-compose.yml
 
-The Langflow project includes a [`docker-compose.yml`](https://github.com/langflow-ai/langflow/blob/main/docker_example/docker-compose.yml) file  for quick deployment with PostgreSQL.
+The Langflow project includes a [docker-compose.yml](https://github.com/langflow-ai/langflow/blob/main/docker_example/docker-compose.yml) file for quick deployment with PostgreSQL.
 
-This configuration launches Langflow and PostgreSQL containers, with Langflow pre-configured to use the PostgreSQL database. Customize the database credentials as needed.
+This configuration launches Langflow and PostgreSQL containers in the same Docker network, ensuring proper connectivity between services. It also sets up persistent volumes for both Langflow and PostgreSQL data.
 
 To start the services, navigate to the `/docker_example` directory, and then run `docker-compose up`.
 
 ```yaml
 services:
   langflow:
-    image: langflow-ai/langflow:latest
-    environment:
-      - LANGFLOW_DATABASE_URL=postgresql://user:password@postgres:5432/langflow
+    image: langflowai/langflow:latest    # or another version tag on https://hub.docker.com/r/langflowai/langflow
+    pull_policy: always                   # set to 'always' when using 'latest' image
+    ports:
+      - "7860:7860"
     depends_on:
       - postgres
+    environment:
+      - LANGFLOW_DATABASE_URL=postgresql://langflow:langflow@postgres:5432/langflow
+      # This variable defines where the logs, file storage, monitor data, and secret keys are stored.
+      - LANGFLOW_CONFIG_DIR=app/langflow
+    volumes:
+      - langflow-data:/app/langflow
 
   postgres:
-    image: postgres:15
+    image: postgres:16
     environment:
-      - POSTGRES_USER=user
-      - POSTGRES_PASSWORD=password
-      - POSTGRES_DB=langflow
+      POSTGRES_USER: langflow
+      POSTGRES_PASSWORD: langflow
+      POSTGRES_DB: langflow
+    ports:
+      - "5432:5432"
+    volumes:
+      - langflow-postgres:/var/lib/postgresql/data
+
+volumes:
+  langflow-postgres:    # Persistent volume for PostgreSQL data
+  langflow-data:        # Persistent volume for Langflow data
+```
+
+:::note
+Docker Compose creates an isolated network for all services defined in the docker-compose.yml file. This ensures that the services can communicate with each other using their service names as hostnames (for example, `postgres` in the database URL). If you were to run PostgreSQL separately using `docker run`, it would be in a different network and Langflow wouldn't be able to connect to it using the service name.
```

Comment:
```suggestion
Docker Compose creates an isolated network for all services defined in the docker-compose.yml file. This ensures that the services can communicate with each other using their service names as hostnames, for example, `postgres` in the database URL. If you were to run PostgreSQL separately using `docker run`, it would be in a different network and Langflow wouldn't be able to connect to it using the service name.
```